Pipfile

```diff
@@ -4,7 +4,7 @@ verify_ssl = true
 name = "pypi"
 
 [packages]
-flask = "*"
+flask = "<3.0.0"
```

Comment:
Are you sure this doesn't work `>3`. According to their set-up they only require `<4`. https://github.com/GoogleCloudPlatform/functions-framework-python/blob/main/setup.py#L53

Reply:
Updated now 👍

Reply:
Also removed wrapt 👍